Gruntfile.js

```diff
@@ -3,6 +3,7 @@ module.exports = function (grunt) {
     // load plugins as needed instead of up front
     require('jit-grunt')(grunt);
     require('phplint').gruntPlugin(grunt);
+    grunt.loadNpmTasks('grunt-contrib-obfuscator');
```

Comment:
I don't think this line is needed, we use jit-grint: https://www.npmjs.com/package/jit-grunt